index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Chapter 4 Links</title>
+  </head>
+  <body>
+    <h1>Welcome to Huong's website!</h1>
+  </body>
+</html>  
```

Comment:
Please add one more empty line here since always have a new line in a file

Reply:
Thank you for your comments. I will fix it